qemu/tests/blockdev_commit_non_existed_node.py

```diff
@@ -15,9 +15,13 @@ def commit_snapshots(self):
         self.device_node = self.get_node_name(device)
         options = ["base-node", "top-node", "speed"]
         arguments = self.params.copy_from_keys(options)
-        arguments["base-node"] = self.params["none_existed_base"]
         device = self.get_node_name(snapshot_tags[-1])
-        arguments["top-node"] = device
+        if self.params.get("none_existed_base"):
+            arguments["base-node"] = self.params["none_existed_base"]
+            arguments["top-node"] = device
+        if self.params.get("none_existed_top"):
+            arguments["base-node"] = self.get_node_name(device)
```

Comment:
From the test log, the base-node is not correct, it should be 'drive_data':

{'execute': 'block-commit', 'arguments': {'base-node': 'drive_drive_sn1', 'top-node': 'sn0', 'device': 'drive_sn1', 'job-id': 'drive_sn1_dkkM'}, 'id': 'VylnU17w'}

Reply:
Yes,thanks for your reminder,updated the code.